.github/workflows/trigger-publish.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - release/v[0-9]+.[0-9]+.[0-9]+ # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags
```

Comment:
First, this doesn't match any current release branches (which are named like `release/v1.14.x`).

Second, how about just make it `release/*`?

Reply:
good catch, yea let's just make * then